stewicombo/globals.py

```diff
@@ -80,9 +80,9 @@ def getInventoriesforFacilityMatches(inventory_dict,facilitymatches,filter_for_L
 def addChemicalMatches(inventories_df):
     #Bring in chemical matches
     chemicalmatches = chemicalmatcher.get_matches_for_StEWI()
-    chemicalmatches = chemicalmatches.drop_duplicates()
+    chemicalmatches = chemicalmatches.drop(columns=['FlowID'])
+    chemicalmatches = chemicalmatches.drop_duplicates(subset=['FlowName','Source'])
```

Comment:
@bl-young can you explain briefly why this subset of drop_duplicates?

Reply:
unlike the other datasets, DMR has instances with the same flow name but different flow ID, which results from the parameter aggregation. These flows end up having the same SRS (as they are the same flow). So if duplicates aren't dropped then records will get duplicated upon merge.